### ISSUE TITLE
Provide ability to specify introductory blurb

### ISSUE THREAD

**Issue by cce** [dkuebric/hubot-release-announce#4]
To humanize these announcements somewhat, it would be nice to be able to provide to Hubot an introductory sentence or two that get included at the top of the email.

**Comment by dkuebric** [comment]
It takes the PR text, so you could include the flavor text there to achieve
this as-is. That's potentially also nice as it will be in github as a
record of the release.

**Comment by danriti** [comment]
@cce I agree with @dankosaur that any and all meta information should be included in the pull release description.
